src/routes/github/github-router.ts

```diff
@@ -33,7 +50,7 @@ subRouter.post("/webhooks",
 // subRouter.use(GithubServerAppMiddleware);
 
 // OAuth Routes
-subRouter.use(jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubOAuthRouter);
```

Comment:
I understand to remove the `jiraSymmetricJwtMidleware`, but why remove the `GithubServerAppMiddleware` here? Even it is added inside the router?

Reply:
Good point, it was originally pulled from the first idea but then didnt put it back afterwards good catch!!

Reply:
LOL, I had a chat with @Harminder84 , we thought it needed to be pulled out because if we leave it here, all routes after this will still run the `GitHubServerAppMiddlware` ? If that's the case, then yes we should pull it out from here and put it into sub router.

ps: also, seems `jiraSymmetricJwtMiddleware` is forgotten to be put inside the `GithubOauthRouter`

Reply:
added it to the /login path but cannot on callback path without casing infinite loops